ci-operator/templates/openshift-azure/cluster-launch-e2e-azure-conformance.yaml

```diff
@@ -0,0 +1,221 @@
+kind: Template
```

Comment:
Why do you have cluster-launch-e2e-azure and cluster-launch-e2e-azure-conformance?

Reply:
The first is meant to run openshift on azure specific e2e tests, the second is meant to run the origin conformance suite on a cluster deployed on azure.